src/component/registry/bpmn-elements-registry.ts

```diff
@@ -133,63 +138,98 @@ export class BpmnElementsRegistry {
+   *
+   * **Note**: If an ID is passed that does not reference an existing BPMN element, no rendering changes are made.
+   *
+   * @example
+   * ```javascript
+   * // Remove all CSS classes from all BPMN elements
+   * bpmnVisualization.bpmnElementsRegistry.removeAllCssClasses();
+   *
+   * // Remove all CSS classes from BPMN elements with ID: activity_1 and activity_2
+   * bpmnVisualization.bpmnElementsRegistry.removeAllCssClasses(['activity_1', 'activity_2']);
+   *
+   * // Remove all CSS classes from BPMN element with ID: task_3
+   * bpmnVisualization.bpmnElementsRegistry.removeAllCssClasses('task_3');
+   * ```
+   *
+   * @param bpmnElementIds The BPMN ID of the element(s) from which to remove all CSS classes.
+   * If no IDs are specified, all CSS classes associated with BPMN elements will be removed.
+   *
+   * @see {@link removeCssClasses} to remove specific classes from a BPMN element.
+   * @since 0.34.0
+   */
+  removeAllCssClasses(bpmnElementIds?: string | string[]): void {
+    if (bpmnElementIds) {
+      ensureIsArray<string>(bpmnElementIds).forEach(bpmnElementId => {
+        const isChanged = this.cssRegistry.removeAllClassNames(bpmnElementId);
+        this.updateCellIfChanged(isChanged, bpmnElementId);
+      });
+    } else {
+      const bpmnIds = this.cssRegistry.getBpmnIds();
+      this.cssRegistry.clear();
+      bpmnIds.forEach(bpmnElementId => this.updateCellIfChanged(true, bpmnElementId));
+    }
+  }
+
+  /**
+   * Toggle one or more CSS classes on one or more BPMN elements.
    *
-   * See also the {@link addCssClasses} or the {@link removeCssClasses} methods.
+   * **Note**: If an ID is passed that does not reference an existing BPMN element, its reference is retained in the registry, but no rendering changes are made.
    *
    * @example
    * ```javascript
-   * // Toggle 'highlight' for BPMN elements with id: activity_1 and activity_2
+   * // Toggle 'highlight' for BPMN elements with ID: activity_1 and activity_2
    * bpmnVisualization.bpmnElementsRegistry.toggleCssClasses(['activity_1', 'activity_2'], 'highlight');
    *
-   * // Toggle 'running' and 'additional-info' for BPMN element with id: task_3
+   * // Toggle 'running' and 'additional-info' for BPMN element with ID: task_3
    * bpmnVisualization.bpmnElementsRegistry.toggleCssClasses('task_3', ['running', 'additional-info']);
    * ```
    *
-   * @param bpmnElementIds The BPMN id of the element(s) where to remove the CSS classes
-   * @param classNames The name of the class(es) to remove from the BPMN element(s)
+   * @param bpmnElementIds The BPMN ID of the element(s) on which to toggle the CSS classes.
+   * @param classNames The name of the class(es) to toggle on the BPMN element(s).
+   *
+   * @see {@link removeCssClasses} to remove specific CSS classes from a BPMN element.
+   * @see {@link removeAllCssClasses} to remove all CSS classes from a BPMN element.
+   * @see {@link addCssClasses} to add CSS classes to a BPMN element.
```

Comment:
**praise:** the JSDoc is correctly display in the API documentation generated by Typedoc

<!--
This works with `toggleCssClasses`. The only difference I detected is that there is no "since" tag in the "toggleCssClasses" JSDoc.
-->

![image](https://user-images.githubusercontent.com/27200110/236212518-1396a463-0ab3-4dff-aed2-54a70efddd78.png)